Jenkinsfile

```diff
@@ -25,47 +35,53 @@ pipeline {
         )
         string(
             name: 'XMOSDOC_VERSION',
-            defaultValue: '6.1.2',
+            defaultValue: 'v6.1.2',
```

Comment:
This should be bumped to 6.1.3 now (released yesterday to fix an xchat issue)

Reply:
Done